### PR TITLE
Immersive paid for label

### PR DESCRIPTION
**Pull request by blongden73**
Adds a label above the headline for the brand which is sponsoring the article. 

picture 371

@regiskuckaertz @NataliaLKB @ScottPainterGNM

### CODE REVIEW

File: common/app/views/fragments/immersiveMainMedia.scala.html [hunk 1]
```diff
@@ -87,9 +87,18 @@
             <div class="gs-container">
                 @if(!isTheMinuteArticle) {
                     <div class="content__main-column">
-                    @fragments.meta.metaInline(page.item)
+                    @if(!isPaidContent){
+                      @fragments.meta.metaInline(page.item)
+                    } else {
+                      <div class="section-label--advertisement">
```
Comment (regiskuckaertz) [inline common/app/views/fragments/immersiveMainMedia.scala.html]: I guess this should be `content__section-label content__section-label--advertisement` eventually

👮

File: common/app/views/fragments/immersiveMainMedia.scala.html [hunk 2]
```diff
@@ -87,9 +87,18 @@
             <div class="gs-container">
                 @if(!isTheMinuteArticle) {
                     <div class="content__main-column">
-                    @fragments.meta.metaInline(page.item)
+                    @if(!isPaidContent){
```
Comment (regiskuckaertz) [inline common/app/views/fragments/immersiveMainMedia.scala.html]: I think this logic should be moved into `metaInline`

Comment (regiskuckaertz) [inline common/app/views/fragments/immersiveMainMedia.scala.html]: in [there](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/meta/metaInline.scala.html#L24), no?

File: static/src/stylesheets/module/content/_article-immersive.scss [hunk 3]
```diff
@@ -237,6 +237,19 @@
 
     /* Meta
     ========================================================================== */
+    .section-label--advertisement {
+        @include fs-textSans(3);
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content/_article-immersive.scss]: Seems silly to use this mixin, and then overwrite the `font-size` and `line-height`. Can you either add another line to this mixin [here](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss#L42) and name it 7, or don't use this mixin, and use this one instead: https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss#L114

File: common/app/views/fragments/meta/metaInline.scala.html [hunk 4]
```diff
@@ -21,6 +21,14 @@
         }
     }
 
+    @if(item.content.isPaidContent) {
```
Comment (regiskuckaertz) [inline common/app/views/fragments/meta/metaInline.scala.html]: Just make sure the item is also an immersive and you should be good to go:
`@if(item.content.isPaidContent && item.content.isImmersive)`

File: common/app/views/fragments/meta/metaInline.scala.html [hunk 5]
```diff
@@ -21,6 +21,14 @@
...
+      <div class="content__section-label--advertisement">
+        <a class="content__section-label__link content__section-label__link--advertisement" href="@LinkTo {/@item.content.sectionLabelLink}">
+          @Html(Localisation(item.content.sectionLabelName))
+        </a>
+      </div>
+    }
```
Comment (regiskuckaertz) [inline common/app/views/fragments/meta/metaInline.scala.html]: To be on the safe side, I'd set the next condition as an alternate:

`} else if(!(item.content.isImmersive && item.content.tags.isArticle)) {`

**Review by regiskuckaertz — APPROVED** [review]
YEAH!

**Pull request merged**